PiwikTracker.php

```diff
@@ -1070,11 +1107,17 @@ public function getUrlTrackAction($actionUrl, $actionType)
      *
      * Allowed only for Super User, must be used along with setTokenAuth()
      * @see setTokenAuth()
-     * @param string $dateTime Date with the format 'Y-m-d H:i:s', or a UNIX timestamp
+     * @param string|DateTimeInterface $dateTime Date with the format 'Y-m-d H:i:s', or a UNIX timestamp, or DateTimeInterface object
+     * @return $this
      */
     public function setForceVisitDateTime($dateTime)
     {
-        $this->forcedDatetime = $dateTime;
+        if ($dateTime instanceof DateTimeInterface) {
```

Comment:
As `DateTimeInterface ` was introduced in PHP 5.5 I presume this would fail on PHP 5.3 etc so we can't really merge this change. You could maybe check if the interface exists then it could be ok. Any other thoughts on this?